.coderabbit.yaml

```
@@ -0,0 +1,886 @@
+        "Script files must use Bun-first patterns: Bun.$ for shell commands, Bun.file() for file operations, proper
+        error handling and logging. Include comprehensive JSDoc documentation and follow enterprise security patterns."
+
+    - path: "docs/*.md"
+      instructions:
+        "Documentation files must follow markdown standards: proper heading hierarchy, code blocks with language
+        specification, consistent formatting, comprehensive coverage of topics. Include security considerations and
+        compliance notes where relevant."
+
+    - path: "**/*.{ts,tsx,js,jsx}"
+      instructions: >
+        Use Bun's native APIs in app code (Bun.serve, bun:sqlite, Bun.redis, Bun.sql, Bun.file, Bun.$, Bun.password).  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)